src/components/RefinementList/RefinementList.tsx

```diff
@@ -302,18 +311,17 @@ class RefinementList<TTemplates extends Templates> extends Component<
       this.props.searchIsAlwaysActive !== true &&
       !(this.props.isFromSearch || !this.props.hasExhaustiveItems);
 
-    const templates = this.props.searchBoxTemplateProps
-      ? this.props.searchBoxTemplateProps.templates
-      : undefined;
-
     const searchBox = this.props.searchFacetValues && (
       <div className={this.props.cssClasses.searchBox}>
         <SearchBox
           ref={this.searchBox}
           placeholder={this.props.searchPlaceholder}
           disabled={shouldDisableSearchBox}
           cssClasses={this.props.cssClasses.searchable!}
-          templates={templates}
+          templates={{
+            ...defaultSearchBoxTemplates,
```

Comment:
I'm not sure, but changing the defaults could be a breaking change, what are the templates exactly?

Don't they get defaulted again in the searchbox component? 

Thanks for checking!